src/main/java/persistence/sql/ddl/generator/CreateDDLQueryGenerator.java

```diff
@@ -0,0 +1,50 @@
+package persistence.sql.ddl.generator;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import persistence.sql.ddl.ColumnAnnotationHelper;
+import persistence.sql.ddl.dialect.ColumnType;
+import persistence.sql.ddl.exception.RequiredAnnotationException;
+import persistence.sql.ddl.schema.ColumnMeta;
+import persistence.sql.ddl.schema.TableMeta;
+
+public class CreateDDLQueryGenerator {
+
+    private final ColumnType columnType;
+    private static final String COMMA = ",";
+    private static final String SPACE = " ";
+    public static final String CREATE_TABLE_FORMAT = "CREATE TABLE %s (%s);";
+
+    public CreateDDLQueryGenerator(ColumnType columnType) {
+        this.columnType = columnType;
+    }
+
+    public String create(Class<?> entityClazz) {
+        return CREATE_TABLE_FORMAT.formatted(
+            TableMeta.of(entityClazz).getTableName(),
+            appendField(entityClazz)
+        );
+    }
+
+    private String appendField(Class<?> entityClazz) {
+        final List<Field> createDDlTargetFieldList = Arrays.stream(entityClazz.getDeclaredFields())
```

Comment:
자료구조의 이름을 그대로 변수의 이름으로 가져가는 것은 지양하는게 좋습니다.
List가 아닌 Set으로 변경되어야한다면 변수명도 함께 변경해주어야 하기 때문에 복수형을 사용해보시는 것도 좋을 것 같습니다 :)

```suggestion
        final List<Field> createDDlTargetFields = Arrays.stream(entityClazz.getDeclaredFields())
```

Reply:
해당 부분도 주로 사용하는 컨벤션이기도 했는데 자료형을 밝혀주므로써 생기는 장점도 있다고 생각하는데 이 부분은 어떻게 생각하시나요?

예를 들면 List는 순서를 보장하지만 Set는 순서를 보장하지 않기 때문에 "s"로 복수만 의미할 경우 내가 원하는 것은 순서를 보장한 자료구조이지만 기대값은 순서를 보장하지않는 경우가 발생할 수도 있는 자료구조간의 차이를 바로 인식할 수 있도록 명시적으로 밝혀주는 것도 좋다고 생각합니다!

Reply:
기륭님이 이야기해주신 장점 부분도 충분히 공감이 가는 것 같습니다 👍 
개인적으로는 코드의 가독성과 변경에 대해 중점을 두고 있기 때문에 자료구조의 변수명을 지양하지만,
자료구조간의 차이로 인한 명시적으로 밝혀주는 것 또한 좋은 것 같네요 :)